Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -26,33 +26,21 @@
 namespace Audio {
 
 FlacLoaderPlugin::FlacLoaderPlugin(StringView path)
-    : m_file(Core::File::construct(path))
+    : m_path(path)
 {
-    if (!m_file->open(Core::OpenMode::ReadOnly)) {
-        m_error = LoaderError { String::formatted("Can't open file: {}", m_file->error_string()) };
-        return;
-    }
-
-    auto maybe_stream = Core::Stream::BufferedFile::create(MUST(Core::Stream::File::open(path, Core::Stream::OpenMode::Read)), FLAC_BUFFER_SIZE);
-    if (maybe_stream.is_error())
-        m_error = LoaderError { "Can't open file stream" };
-    else
-        m_stream = maybe_stream.release_value();
 }
 
-FlacLoaderPlugin::FlacLoaderPlugin(Bytes& buffer)
+FlacLoaderPlugin::FlacLoaderPlugin(Bytes const& buffer)
```

Comment:
Just pass it by value, no need to pass a span by const reference (same as StringView)